src/OrchardCore.Modules/OrchardCore.Users/Views/UserActionsMenu.cshtml

```diff
@@ -14,7 +14,14 @@
 @if (canEdit)
 {
     <li>
-        <a asp-action="EditPassword" asp-route-id="@Model.User.UserId" class="dropdown-item">@T["Change password"]</a>
+        @if (isCurrentUser)
```

Comment:
@hishamco why was this done? This just forces the current user (like an admin) to use a different process. But the backend process does not enforce this logic, If this is the process that we want to follow, we should enforce this logic via the controller.

Reply:
Let me bring that code into the edit instead of reverting the PR

Reply:
@MikeAlhayek there's a `ResetPassword` feature to change password, could you please refer me to that feature that you are talking about?

Reply:
Resetting is something different than changing the password, right?

Reply:
Resetting is something different than changing the password, right?

Reply:
Yes. Resetting password is done using front end. Changing password is an admin changing the password for any user